### PR TITLE
LPS-176442 Fix errors in Tear Down

### PR DESCRIPTION
**Pull request by magjed4289**
https://issues.liferay.com/browse/LPS-176442

### CODE REVIEW

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

**Review by Hijky — APPROVED** [review]

File: modules/apps/object/object-rest-test/src/testFunctional/tests/AdvancedObjectAPI.testcase [hunk 1]
```diff
@@ -34,12 +34,10 @@ definition {
 	}
 
 	tearDown {
-		ObjectAdmin.deleteObjectViaAPI(objectName = "Manager");
-
-		ObjectAdmin.deleteObjectViaAPI(objectName = "Employee");
-
 		var testPortalInstance = PropsUtil.get("test.portal.instance");
 
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
```
Comment (yanan) [inline modules/apps/object/object-rest-test/src/testFunctional/tests/AdvancedObjectAPI.testcase:39]: :+1:

File: portal-web/test/functional/com/liferay/portalweb/macros/json/user/JSONUserGetter.macro [hunk 2]
```diff
@@ -57,11 +50,11 @@ definition {
 	macro getEmailAddressNoSelenium {
 		Variables.assertDefined(parameterList = ${userId});
 
-		var curl = JSONUserGetter._getUserByIdCurlNoSelenium(
-			authenticationMethod = ${authenticationMethod},
-			userId = ${userId});
+		var curl = JSONUserGetter._getUserByIdCurlNoSelenium(userId = ${userId});
 
-		var userEmailAddress = JSONCurlUtil.get(${curl}, "$['emailAddress']");
+		var response = JSONCurlUtil.get(${curl});
```
Comment (yanan) [inline portal-web/test/functional/com/liferay/portalweb/macros/json/user/JSONUserGetter.macro:55]: Seeing here using two lines to get userEmailAddress, wondering are we going to reduce using JOSNCurlUtil.get(${curl},"$..emailAddress")?

**Review by yanan — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/macros/json/user/JSONUserGetter.macro (hunk 2 above)
Comment (magjed4289) [reply portal-web/test/functional/com/liferay/portalweb/macros/json/user/JSONUserGetter.macro:55]: this way it works, seems like the $['emailAddress']" part was a problem, I will refactor thanks!

**Comment by magjed4289** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
ci:forward

**Comment by magjed4289** [comment]
@Hijky once this is merged into master you can rebase and commit https://github.com/liferay-headless/liferay-portal/pull/1058, thanks!

**Pull request closed without merging**